server/start.go

```diff
@@ -149,14 +149,17 @@ which accepts a path for the resulting pprof file.
 			serverCtx.Logger.Info("Unlocking keyring")
 
 			// fire unlock precess for keyring
-			keyringBackend, _ := cmd.Flags().GetString(flags.FlagKeyringBackend)
```

Comment:
seems we hasn't support keyring-backend flag in start cmd before as well

Reply:
good catch, added now

Reply:
ethermint add [global](https://github.com/crypto-org-chain/ethermint/blob/develop/server/flags/flags.go#L95) but cronos doesn't yet

Reply:
```
$ ./build/ethermintd start --help|grep keyring
      --keyring-backend string   Select keyring's backend (default "os")
```

hmm, it seems it's there already before, why you say that?

Reply:
> ethermint add [global](https://github.com/crypto-org-chain/ethermint/blob/develop/server/flags/flags.go#L95) but cronos doesn't yet

I see, cronos can call `AddGlobalFlags` as well then.